life_expectancy/clean_data/utils/countries.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
+from typing import List
+
+class Country(Enum):
+    """Enum class representing a country"""
+    AT = "AT"
+    BE = "BE"
+    BG = "BG"
+    CH = "CH"
+    CY = "CY"
+    CZ = "CZ"
+    DK = "DK"
+    EE = "EE"
+    EL = "EL"
+    ES = "ES"
+    EU27_2020 = "EU27_2020"
+    FI = "FI"
+    FR = "FR"
+    HR = "HR"
+    HU = "HU"
+    IS = "IS"
+    IT = "IT"
+    LI = "LI"
+    LT = "LT"
+    LU = "LU"
+    LV = "LV"
+    MT = "MT"
+    NL = "NL"
+    NO = "NO"
+    PL = "PL"
+    PT = "PT"
+    RO = "RO"
+    SE = "SE"
+    SI = "SI"
+    SK = "SK"
+    DE = "DE"
+    DE_TOT = "DE_TOT"
+    AL = "AL"
+    EA18 = "EA18"
+    EA19 = "EA19"
+    EFTA = "EFTA"
+    IE = "IE"
+    ME = "ME"
+    MK = "MK"
+    RS = "RS"
+    AM = "AM"
+    AZ = "AZ"
+    GE = "GE"
+    TR = "TR"
+    UA = "UA"
+    BYAAAA = "BY"
+    EEA30_2007 = "EEA30_2007"
+    EEA3 = "EEA3"
+
+    @classmethod
+    def get_countries(cls) -> List[str]:
+        """Get a list with all the countries"""
+        all_countries = set()
```

Comment:
You could have used the decorator @unique before the class definition. In this way, you will not need a set to ensure no repetitive countries.
I believe that you could also have just return a list comprehension (something like `sorted([country.value for country in cls if len(country.value)==2])`). But this might be a "matter of personal taste".

Reply:
Thanks for the suggestion! 🙂 I'll change my code – your idea is much less verbose than mine!

Reply:
Applied suggestion in commit [091f593](https://github.com/2010b9/learning-pod-foundations-assignments/pull/7/commits/091f59393e9d8f921271f4dee61636b1df85297a)